### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by TheJoeFin** [TheJoeFin/Windows10-Community#103]
- Text Cursor Indicator Preview: Changes to the text cursor indicator can now be viewed on the fly in a new preview pane when changing the size or color in the Text cursor page in Ease of Access.
- Magnifier reading now supports 3 new ways to start reading: Press the play button to start reading from the top of content, or selected text if text was selected, or press the read from here button then click where you want to start reading. Note that this experience works with apps with large content like Edge, Word, Mail, etc. Known issue: Play from top may not work at times and you will hear “cannot read from here.”
- Narrator gives page summaries and popular links: In Build 18894,we released a quick summary with Narrator + S that will give you headings, links, and landmarks. Pressing Narrator + S twice quickly will open a dialog window with the number of headings, landmarks, and links and a list of popular links. Popular links are the most commonly used links on the website, and you can navigate directly to those links from the page summary.
- New Narrator Sound Scheme: We’ve revamped Narrator sounds to be more functional and responsive. As part of that effort, we added sounds for common actions, such as when you land on a link or when you turn on or off scan mode. To manage sounds for common actions, go to Narrator settings.

**Comment by TheJoeFin** [comment]
- Narrator will now automatically start reading webpages and emails: Do you want to hear the most valuable content faster? Narrator now automatically starts reading webpages when they are loaded. Narrator will start reading at the main landmark, if there is one, or will fall back to a reasonable paragraph. Narrator will also start automatically reading emails when they are opened in Outlook and the inbox Mail applications. If you find experiences that aren’t working well, please send us feedback.
- The Magnifier UI now works with dark theme and text sizing options. The image below shows two Magnifier windows. The top one is in Light theme and default text size; the bottom one is in Dark theme and the text is scaled at 125%.

**Comment by TheJoeFin** [comment]
- **Table reading improvements:** Narrator is now more efficient when reading tables. Header information is not repeated when navigating within the same row or column. Entering and exiting tables is also less verbose.
- **Narrator web page summary:** There’s a new command in Narrator to give a webpage summary! (Narrator + S). Currently this command will give information about hyperlinks, landmarks and headings.
- **Magnifier text cursor setting:** Windows Magnifier has a new ability to keep the text cursor in the center of the screen making it easier and smoother to type. Centered on the screen is on by default and can be changed in the Magnifier settings.
